src/runtime_src/core/tools/common/ReportAie.cpp

```diff
@@ -200,77 +200,77 @@ populate_aie_core(boost::property_tree::ptree _pt,boost::property_tree::ptree& t
     tile.put("row", pt.get<uint32_t>("row"));
 
     std::string status;
-    for (auto& node: pt.get_child("core.Status", empty_pt)) {
+    for (auto& node: pt.get_child("core.status", empty_pt)) {

```

Comment:
Note: This comment applied to ALL of the schema name changes.

What the name change?

I ask for the last time we chatted regarding the names, it was mentioned that these names followed the documentation and they could not be changed.  Has the documentation changed?

Reply:
this is intermediate schema which has changes it doesn't affect output.
we pushed changes base on spec given by aie driver but when actual aie driver changes came, they differ from spec.